content/en/docs/setup/install/helm/test.sh

```diff
@@ -27,7 +27,7 @@ function rewrite-repo() {
   cmd="$(echo "${cmd}" | sed 's|istio/base|manifests/charts/base|')"
   cmd="$(echo "${cmd}" | sed 's|istio/istiod|manifests/charts/istio-control/istio-discovery|')"
   cmd="$(echo "${cmd}" | sed 's|istio/gateway|manifests/charts/gateway|')"
-  eval "${cmd}"
+  eval "${cmd} --set global.tag=${ISTIO_IMAGE_VERSION=SHOULD_BE_SET}.${ISTIO_LONG_SHA=latest}"
```

Comment:
I need to set default values for these, or override the `-u`. These values are set in the Makefile. If the values aren't set, the defaults will lead to the test failing which is OK as there aren't really a good set of defaults.